### PR TITLE
Fixing bug with image resizing

### PR DESCRIPTION
**Pull request by simonstaton**

### CODE REVIEW

**Comment by osavchenko** [comment]
Looks good :+1:

**Comment by BernardoSilva** [comment]
Can you add a test to prove the bug is fixed?

File: package.json [hunk 1]
```diff
@@ -60,6 +61,6 @@
     "Partner Marketing <supp.acc@partnermarketing.com>"
   ],
   "scripts": {
-    "test": "eslint ./src && karma start ./tests/unit/karma.conf.js"
+    "test": "karma start ./tests/unit/karma.conf.js"
```
Comment (osavchenko) [inline package.json]: why you remove eslint check?

**Comment by osavchenko** [comment]
:+1:

**Pull request merged**